finalPrep/modZeroHero.js

```diff
@@ -56,16 +120,26 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
 
+var scaryMonster = {
+  name: 'Catzilla',
+  smell: 'sulferous',
+  weight: 20000,
+  citiesDestroyed: ['Tokyo', 'San Francisco', 'New York'],
+  luckyNumbers: [14, 42, 7],
+  address: {number: '1234', street: 'Main Street', state: 'SD', zip: '00001' }
```

Comment:
Generally it's a good idea to format objects with key/value pairs on their own respective lines, like so:

```
address: {
    number: '1234',
    street: 'Main Street',
    state: 'SD',
    zip: '00001'
}
```